demo/helm/rook-operator/README.md

```diff
@@ -15,7 +15,7 @@ Alternatively, to deploy from a local checkout of the rook codebase (until the r
 
 ```console
 $ cd demo/helm/rook-operator
-$ helm install --name rook-operator --namespace rook .
+$ helm install --name rook-operator --namespace default .
```

Comment:
ok, reverting back to rook namespace, smoke test passed on both rook and default namespaces.